.circleci/TestSplitting.yml

```diff
@@ -0,0 +1,53 @@
+version: 2.1
+orbs:
+  matlab: mathworks/matlab@1
+  win: circleci/windows@5
+
+# Define the execution environments for different operating systems
```

Comment:
operating systems --> platforms (if you decide to replace OS with platform below)